python/looker_sdk/rtl/transport.py

```diff
@@ -44,14 +44,23 @@ class HttpMethod(enum.Enum):
     HEAD = 7
 
 
+TConnectTimeout = int
+TReadTimeout = int
+TConnectAndReadTimeout = int
+TConnectAndReadTimeoutSeconds = Union[
+    Tuple[TConnectTimeout, TReadTimeout], TConnectAndReadTimeout
+]
+
+
 @attr.s(auto_attribs=True, kw_only=True)
 class TransportSettings:
     """Basic transport settings.
     """
 
-    base_url: str
+    base_url: str = ""
     api_version: str = "3.1"
     verify_ssl: bool = True
+    timeout: TConnectAndReadTimeout = (1800, 120)
```

Comment:
now that I see it, I feel like `1800` is way too long to wait to establish a connection. I say lets keep them the same number. Unless you feel strongly otherwise then we can leave it

```suggestion
    timeout: TConnectAndReadTimeout = (120, 120)
```